### PR TITLE
Update 'scheduled-function' example #42

### PR DESCRIPTION
**Pull request by amongil**
This should fix issue #42

### CODE REVIEW

**Comment by sam-goodwin** [comment]
Hey! Thanks for taking the initiative to fix the example. I think you might have the change the wrong way around though. Have you re-based since this change? https://github.com/sam-goodwin/punchcard/pull/26

File: examples/lib/scheduled-function.ts [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
+import punchcard = require('punchcard');
+import { string, integer } from 'punchcard';
 
 const app = new cdk.App();
 export default app;
 
 const stack = new cdk.Stack(app, 'scheduled-function-example');
-
-const table = new DynamoDB.HashTable(stack, 'my-table', {
+const table = new punchcard.HashTable(stack, 'my-table', {
```
Comment (sam-goodwin) [inline examples/lib/scheduled-function.ts]: I think you have an old version, I moved everything into service-namespaces in https://github.com/sam-goodwin/punchcard/pull/26

**Comment by sam-goodwin** [comment]
Oh, I actually know what's happening. I haven't released the changes to NPM yet. I will release them in version 0.7.0 today. Stay tuned for that.

**Comment by sam-goodwin** [comment]
If you want to run the examples, make sure you clone the repo and run the following to build the project:

```
lerna bootstrap
lerna run build
lerna run test
cd examples
cdk deploy -a ./lib/scheduled-function.js
```

This will build against the local code instead of NPM

**Comment by amongil** [comment]
Great stuff Sam, thanks for the help. Will work this way, then. Closing.

**Pull request closed without merging**